src/iop/highlights.c

```diff
@@ -546,7 +546,7 @@ static void process_lch_bayer(dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *pie
 #ifdef _OPENMP
 #pragma omp parallel for default(none) \
   dt_omp_firstprivate(clip, filters, ivoid, ovoid, roi_out) \
-  schedule(dynamic)
+  schedule(static)
 #endif
   for(int j = 0; j < roi_out->height; j++)
   {
```

Comment:
that would be better if the loops could be perfectly nested too, so we could pass `collapse(2)`

Reply:
Done.